pyproject.toml

```diff
@@ -5,21 +5,22 @@ description = ""
 authors = ["Kaj Siebert <kaj@k-si.com>", "Matthew Pugh <matthew.pugh@socialfinance.org.uk"]
 
 [tool.poetry.dependencies]
-python = ">=3.8,<4"
+python = ">=3.9,<4"
 Django = "^4.2.16"
-twilio = "^7.2.0"
-django-split-settings = "^1.1.0"
+twilio = "^9.4.1"
+django-split-settings = "^1.3.2"
 python-dotenv = "^0.19.1"
 django-ipware = "^4.0.0"
 django-heroku = "^0.3.1"
-gunicorn = "^20.1.0"
+gunicorn = "^23.0.0"
 msgraphy = "^0.2.1"
-slack-sdk = "^3.11.2"
-sentry-sdk = "^1.4.3"
+slack-sdk = "^3.34.0"
+sentry-sdk = "^2.19.2"
 django-q = {extras = ["sentry"], version = "^1.3.9"}
 croniter = "^1.0.15"
 tablib = {extras = ["xlsx", "cli"], version = "^3.2.1"}
-django-extensions = "^3.1.5"
+django-extensions = "^3.2.3"
+setuptools = "^75.6.0"
```

Comment:
Needed for one of the updates?

Reply:
Yes, there are some security updates.  The django extensions update fixes a security issue. Adding setup tools was because some of the updates need this explicitly defined as it was a secondary import before